bcda/api.go

```diff
@@ -54,6 +54,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const JobTimeout = time.Hour * 24
```

Comment:
Could we use the `ARCHIVE_THRESHOLD_HR` for this if it's available (default to 24 hours if not) to match the `archive-job-files` CLI action?

Reply:
I added a GetJobTimeout() method that is controlled by that environment variable.  I thought the value was passed into the job as a CLI.  This is a much better solution.  I also updated 710 with the same logic.